src/settings.rs

```diff
@@ -14,6 +14,7 @@ static GIGABYTE: u32 = MEGABYTE * 1_000;
 static PREFIX: &str = "skeleton";
 
 #[derive(Clone, Debug, Deserialize)]
+#[serde(default)]
```

Comment:
FYI: this ends up being the solution for the [dual default settings nit](https://github.com/mozilla-services/syncstorage-rs/issues/253) (originally the config crate added another method to `Config` to fix this, then they reverted it after finding serde can solve it)

Reply:
oh, Yay! I was wondering when that would land.